Summarization.py

```diff
@@ -11,155 +11,168 @@
 import re
 import docx
 from nltk.tokenize import sent_tokenize
+import argparse 
+import sys
 #pip uninstall docx
 #pip install python-docx
-from docx import Document
-document = Document()
-document = Document('C:/Users/wb550776/Downloads/Using ML in Evaluative Synthesis May 31 2019.docx')
+#from docx import Document
+#document = Document()
+#document = Document('C:/Users/wb550776/Downloads/Using ML in Evaluative Synthesis May 31 2019.docx')
 
+
+def main():
+    parser= argparse.ArgumentParser()
+    parser.argument("/Users/minaekramnia/Documents/CV Cover Letter/Mina Ekramnia_CoverLetter_Educ.docx")
+    args = parser.parse_args()
+    #the output comes to the council itself
+    sys.stdout.write(str(Summerization(args)))
 #Open the article
-def read_article(filename):
-    file = open(r'C:/Users/wb550776/Downloads/10144-Ghana-ELAC EvNote.docx', encoding='utf-8')
-    #file2 = open(r'C:/Users/wb550776/Downloads/README.txt')
-    filedata = file.readlines()
-    article = filedata[0].split(". ")
+    
+def Summerization(args):

```

Comment:
nitpick, but the standard convention is to lowercase function names

Reply:
also, you should be explicit about what the input variables are. Reading this I have no idea what `args` is supposed to be